### PR TITLE
Add two columns footer

### LINKED ISSUE #10: Create footer for webpage
The footer should display the standard branding and useful links area
The useful links area will display: 

COLUMN 1
- Despre proiect 
- Guvernul Romaniei
- Departamentul pentru Situații de Urgență
- Ministerul Sanatatii 

COLUMN 2
- Știri oficiale
- Ce trebuie să fac
- Date la zi
- RoHelp
- Code for Romania

background color: #00478F

### PR DESCRIPTION
**Pull request by Anca07**
Closes #10 

Add a two column footer for the webpage

image

### CODE REVIEW

File: src/components/Footer/index.js [hunk 1]
```diff
@@ -15,6 +15,32 @@ const FooterWrapper = () => {
...
+            Guvernul României
+          </a>
+        </FooterLinkItem>
+        <FooterLinkItem>
+          <a
+            href="http://www.dsu.mai.gov.ro/"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Departamentul pentru Situații de Urgență
```
Comment (tudoramariei) [inline src/components/Footer/index.js]: this will need to be broken in two lines
```
Departamentul pentru 
Situații de Urgență
```

**Comment by tudoramariei** [comment]
One more comment, please make sure your git is configured on your computer so that your commits are properly signed. Right now we wouldn't be able to find the committer on GitHub

File: src/components/Footer/index.js (hunk 1 above)
Comment (Anca07) [reply src/components/Footer/index.js]: Done, thanks!

Sorry about the git configuration, I have another account configured on the machine and I forgot to switch users.

**Comment by tudoramariei** [comment]
I'll also ask you to squash your changes into one commit, to make sure that your signed commit is the only one visible

**Pull request merged**